cmd/scollector/collectors/apache_mod_info.go

```diff
@@ -0,0 +1,121 @@
+package collectors
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"golang.org/x/net/html"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+func init() {
+	collectors = append(collectors, &IntervalCollector{F: c_apache_mod_info, Interval: time.Minute * 30})
```

Comment:
Add an `Enable: enableURL(apacheModInfoURL)` to the struct, which will only run this if that URL returns a 200. See https://github.com/bosun-monitor/bosun/blob/21d1e9d4e22158ac031c030918d6affad9efa967/cmd/scollector/collectors/opentsdb.go#L12 for how we do that. Note use of the const for the URL.